tests/test_csc.py

```diff
@@ -74,15 +79,28 @@ async def test_telemetry(self):
                 m_ldco48_temperature=19,
                 m_ldco48_temperature_2=19,
             )
+            self.csc.simulator.device.do_set_m_cpu800_18_power("FAULT")
+            await self.assert_next_sample(
+                topic=self.remote.evt_summaryState,
+                summaryState=salobj.State.FAULT,
+                flush=True,
+            )
 
     async def test_change_wavelength(self):
         async with self.make_csc(initial_state=salobj.State.ENABLED, simulation_mode=1):
             await self.remote.cmd_changeWavelength.set_start(
                 wavelength=700, timeout=STD_TIMEOUT
             )
+            await self.assert_next_sample(
+                topic=self.remote.evt_wavelengthChanged, wavelength=700
+            )
             await self.assert_next_sample(
                 topic=self.remote.tel_wavelength, wavelength=700, flush=True
             )
+            with pytest.raises(salobj.AckError):
+                await self.remote.cmd_changeWavelength.set_start(
+                    wavelength=1200, timeout=STD_TIMEOUT
```

Comment:
Please add a code comment explaining why this should raise. I suspect it's out of range?

The test would be a bit clearer if it actually used the range from the CSC itself (or, rather, the hardware model), if that can be read from configuration or something, e.g. `wavelength= max_wavelength + 0.1`

Please consider testing min wavelength as well.